### ISSUE TITLE
Bug: No query log on getting parser error

### ISSUE THREAD

**Issue by flaneur2020** [datafuselabs/databend#10882]
**Summary**

I can get the following error on parser error in logs, but can not get the original SQL which makes this parser error:

![Screen Shot 2023-03-31 at 19 02 59](https://user-images.githubusercontent.com/129800/229103666-2f564cac-d28d-4e4c-bb58-5cc6f83b98fd.png)

**Comment by sundy-li** [comment]
Err message will be truncated by a limit length string.

The original SQL may be very long, which truncates the error msg.

**Comment by flaneur2020** [comment]
@sundy-li oh, it seems the query_log missed the log item on queries with bad syntax. I can not find the relevant log item in this case.

**Comment by sundy-li** [comment]
Yes, query log only records query with correct syntax, it's inside pipeline.

**Comment by KKould** [comment]
fixed: https://github.com/databendlabs/databend/pull/15077
